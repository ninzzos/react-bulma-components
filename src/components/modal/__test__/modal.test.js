import React from 'react';
import renderer from 'react-test-renderer';
import Modal from '..';

describe('Modal component', () => {
  it('Should Exist', () => {
    expect(Modal).toMatchSnapshot();
  });
  it('Should have modal classname', () => {
    const component = renderer.create(
      <Modal>Modal content</Modal>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
