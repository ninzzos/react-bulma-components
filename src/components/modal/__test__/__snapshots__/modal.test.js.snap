// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal component Should Exist 1`] = `[Function]`;

exports[`Modal component Should have modal classname 1`] = `
<div
  className="modal is-active"
>
  <div
    className="modal-background"
  />
  <div
    className="modal-content"
  >
    Modal content
  </div>
  <button
    aria-label="close"
    className="modal-close is-large"
  />
</div>
`;
